### PR TITLE
Finish-mobile-portfolio

### PR DESCRIPTION
**Pull request by ariful04**
In this branch I did:
- [ ] Add HTML file.
- [ ] Add CSS file.
- [ ] Check Linter error
- [ ] Add work section
- [ ] Add About myself section
I try to design as possible as Figma template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    </div>
+     <p class="project-description">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+     </p>
+     <ul class="p-languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">java script</li>
+     </ul>
```
Comment (franclobo) [inline index.html]: - [ ] Kindly use capital letters to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
```
Comment (franclobo) [inline index.html]: - [ ] Kindly use capital letters to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
```
Comment (franclobo) [inline index.html]: - [ ] Kindly use capital letters to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
```
Comment (franclobo) [inline index.html]: - [ ] Kindly use capital letters to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+                <a href="#"><img class="social-icon" src="/images/medium.svg" alt="Medium logo" /></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/github.svg" alt="Github Logo" /></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/angellist.svg" alt="Angellist" /></a>
+            </li>
+        </ul>
+        <button class="resume-buton" type="button">Get My Resume</button>
```
Comment (franclobo) [inline index.html]: - [ ] Please 🙏🏽 use the lower letters in the corresponding words to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @ariful04, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I'm sure that your project is almost complete, but there are two things to correct: 1. The left margin and 2. The path of the images needs a dot at the beginning to avoid broken links, well done using the alt attribute to describe each image. 

https://github.com/ariful04/Myportfolio/assets/58642949/8d991977-99ea-4441-858c-055e2aea1d1a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ariful04** [comment]
Hello @franclobo, thank you so much for fast review and give nice descriptive feedback. I update my html file as your requirement, in before i forget to push image folder so image link are broken, Now i think alright please recheck.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

**Hi @ariful04 ,**

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented previous suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**